srcpkgs/hunspell-ro_RO/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'hunspell-ro_RO'
+pkgname=hunspell-ro_RO
+version=3.3.10
+revision=1
+wrksrc=hunspell-ro.${version}
+short_desc="Romanian ro_RO dictionary for hunspell"
```

Comment:
Does it need `ro_RO` in the description?

Reply:
Not really, I just saw that some other dictionaries had it and decided to add it. I suppose it's unnecessary.